app/views/list/index.html.slim

```diff
@@ -0,0 +1,44 @@
+html[lang="en"]
+  head
+    meta[charset="utf-8"]
+    title
+      | Wishlists
+  body
+    .container-fluid
+      .row
+        .col-xs-12
+          .wishlist-header
+            | Wishlists
+          .wishlist-page
+            - if @lists.any?
+            .table-responsive
+              table#catalog-table.wishlist-table.table.table-hover
+                thead
+                  th
+                  th
+                  th
+                    | Title
+                  th
+                    | Description
+                  th
+                    | Created At
+                  th
+                    | Updated At
+                tbody
+                - for list in @lists
+                  tr.request
+                    td.request-checkbox
+                      input[type="checkbox"]
+                    td.request-edit
+                      a[href="#"]
+                        | Edit
+                    td = link_to list.title, list_path(list.id)
+                    td = list.description
+                    td = list.created_at
+                    td = list.updated_at
+            .wishlist-remove
+              button.btn.btn-default[type="submit"]
```

Comment:
Remove button should be a different color than the submit button - I suggest `.btn-error`.